src/highrl/utils/action.py

```diff
@@ -1,5 +1,5 @@
 """Implementation of action type for robot"""
 from collections import namedtuple
 
-ActionXY = namedtuple('ActionXY', ['vx', 'vy', 'w']) 
-ActionRot = namedtuple('ActionRot', ['v', 'r'])
+ActionXY = namedtuple("ActionXY", ["vx", "vy", "w"])
```

Comment:
why double quotations?
I mean what is the difference?

Reply:
No difference! Yet the PEP8 convention is make all strings as double quotes.

Reply:
noted